src/components/Layout/Dropdown.astro

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
Oh, one thing I just noticed, let's move this file up a folder into the `src/components` directory directly since it's not exclusive to layouts?

Reply:
Done!